ddtrace/llmobs/_llmobs.py

```diff
@@ -774,6 +777,124 @@ def _tag_metrics(span: Span, metrics: Dict[str, Any]) -> None:
             return
         span.set_tag_str(METRICS, safe_json(metrics))
 
+    @classmethod
+    def submit_evaluation_for(
+        cls,
+        label: str,
+        metric_type: str,
+        value: Union[str, int, float],
+        span: Optional[dict] = None,
+        span_with_tag: Optional[Tuple[str, str]] = None,
```

Comment:
Is it more user friendly to use a tuple (key, value) as an argument for this, or a dictionary ("key": key, "value": value), or two separate arguments `joining_key_name`, `joining_key_value`? I am not 100% sure why but I feel like there could be a better user experience in providing the custom joining key/value.

Reply:
I chose tuple initially since it was the most concise and it also made it clear that a single key/value was accepted.

I see your point though; a dictionary may be better since its
1. consistent with `span` 
2. more verbose/clear, which >>> being concise

Reply:
What do you think? I'm leaning toward `dictionary ("tag_key": key, "tag_value": value)`

Reply:
:+1: dict looks good to me. nit on the argument name, i'd go for `span_with_tag_value` since just `tag` implies (to me) a tag name